packages/cli/test/integration/cli/cli.integration.js

```diff
@@ -44,11 +44,7 @@ describe('cli', () => {
     expect(entries).to.containEql(
       '  lb4 app\n  lb4 extension\n  lb4 controller\n  lb4 datasource\n  ' +
         'lb4 model\n  lb4 repository\n  lb4 service\n  lb4 example\n  ' +
-<<<<<<< HEAD
-        'lb4 openapi\n  lb4 observer',
-=======
-        'lb4 openapi\n  lb4 relation',
->>>>>>> c589f34... feat(cli): add lb4 relation command
```

Comment:
FYI I squashed this fix in the commit because I forgot to remove these diff lines when rebasing.